R/count_watershed_data.R

```diff
@@ -259,6 +265,13 @@ count_watershed_data <- function(data_dir,
           .[["flow"]] %>% min() ->
           historical_flow_dry_m3sec
 
+        # Calculate flow from NHD flowline
+
+        watershed_nhd_flows %>%
+          filter(DVSN_ID %in% watershed) -> nhd_flow
+
+        if_else(length(nhd_flow$.[1]) == 0, historical_runoff_mean_m3sec, nhd_flow$Q0001E[1]) -> nhd_mean_flow_m3sec
```

Comment:
I think `nhd_flow$.[1]` will always result in `NULL` irrespective of whether there's a value available. So you'll end up with `nhd_mean_flow_m3sec` always being reverted to `historical_runoff...`....   Did you run any check to ensure that the nhd flow was actually making it through to the output tables?

Reply:
...maybe it's to do with the version of the flow data I'm using... I downloaded what was on PIC (seemed to be updated today). This still has 0 values rather than NA for `Q0001E`. Did you update this on your own machine, perhaps?

Reply:
@swd-turner hmmm. I did run a couple and it seemed to be working, but I will double check and change to a different method if needed.